lib/tasks/swaggerize.rake

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+if defined?(RSpec)
+  require "rspec/core/rake_task"
+
+  Rake::Task["rswag:specs:swaggerize"].clear
```

Comment:
Can we add a note to `Readme` about generating and re-generating swagger docs?

Reply:
Added here 2fb193f